### ISSUE TITLE
When selecting DMP template - it should say something more generic than funder

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2886]
v.2.2.0 

It has been suggested from a user that currently when a user creates a plan and there is no funder associated with the template but the org has more templates for the user to choose the wording is bit confusing: We found multiple DMP templates corresponding to your funder.
<img width="982" alt="Screenshot 2021-04-23 at 16 47 11" src="https://user-images.githubusercontent.com/47178106/115896786-aefd2a80-a453-11eb-9633-e7a616cc15f0.png">

It would be better to say: We found multiple DMP templates from your organisation or we found multiple DMP templates corresponding to your selected organisations

any thoughts @mariapraetzellis @pherterich

**Comment by mariapraetzellis** [comment]
Yes, this makes sense to me. I think using: "We found multiple DMP templates from your organization" would work here.
